### PR TITLE
Move lists from collections to math and renamed them to ...VectorList

### PR DESCRIPTION
**Pull request by bettinaselig**

### CODE REVIEW

File: source/math/FloatComplexVectorList.ooc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	sum: func -> FloatComplex {
+		result := FloatComplex new()
+		for (i in 0..this _count)
+			result = result + this[i]
+		result
+	}
+	mean: func -> FloatComplex {
+		sum() / this _count
+	}
+	getReal: func -> FloatVectorList {
```
Comment (davidhesselbom) [inline source/math/FloatComplexVectorList.ooc]: No but seriously, shouldn't this be a property?

**Comment by davidhesselbom** [comment]
Needs rebasing.

File: source/math/FloatComplexVectorList.ooc (hunk 1 above)
Comment (bettinaselig) [inline source/math/FloatComplexVectorList.ooc]: Updated.

**Comment by bettinaselig** [comment]
Done.

**Pull request merged**